packages/ember-glimmer/lib/renderer.js

```diff
@@ -367,8 +369,10 @@ class Renderer {
 
     try {
       this._renderRoots();
-    } finally {
+    } catch (e) {
```

Comment:
I'm ok with that, it looks terrible, we could also make it a try/finally between begin/commit